test/helpers/test_provisioner.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require_relative './kubeclient_helper'
+
+class TestProvisioner
+  extend KubeclientHelper
+
+  class << self
+    def prepare_cluster
+      WebMock.allow_net_connect!
+      $stderr.print "Preparing test cluster... "
+      prepare_pv("pv0001")
+      prepare_pv("pv0002")
+      deploy_metric_server
+      $stderr.puts "Done."
+      WebMock.disable_net_connect!
+    end
+
+    def claim_namespace(test_name)
+      test_name = test_name.gsub(/[^-a-z0-9]/, '-').slice(0, 36) # namespace name length must be <= 63 chars
```

Comment:
nitpick: Took me a second to understand why 36 is correct here, could be a tiny bit clearer